src/components/Install/index.tsx

```diff
@@ -54,7 +54,7 @@ export const Install: FunctionComponent = () => {
                         onClick={copy}
                         onKeyDown={copy}
                         role="button"
-                        className={classNames(styles.icon, 'icon-inline ml-4 align-text-top')}
+                        className={classNames(styles.icon, 'ml-4 align-text-top')}
                         tabIndex={0}
                     >
                         <CopyIcon />
```

Comment:
Could you jump into the CopyIcon SVG and change the size of the width and height to `35`? Since the `icon-inline` class was deleted, this changed the size.

Reply:
Sure thing! That's updated now